public/documentation/tutorials/webassembly.md

```diff
@@ -0,0 +1,50 @@
+# Web Assembly
+
+WebAssembly or wasm is a new portable, size- and load-time-efficient format suitable for compilation to the web. It has following advantages:
+
+*   Efficient and fast.
+*   Safe.
+*   Open and debuggable.
+*   Part of the open web platform.
+
+WebAssembly is currently being designed as an open standard by a [W3C Community Group](https://www.w3.org/community/webassembly/) that includes representatives from all major browsers. Now it is available in Crosswalk 22 or later on Android, Windows and Linux. It's also [supported in multiple browsers](https://hacks.mozilla.org/2016/03/a-webassembly-milestone/)
+
+In the following tutorial, you'll learn how to enable WebAssembly in Crosswalk and run the demo [Angrybots](https://webassembly.github.io/demo/) on Windows and Android platforms.
+
+1. Download the [Crosswalk 22 Windows](https://download.01.org/crosswalk/releases/crosswalk/windows/canary/22.52.557.0/crosswalk64-22.52.557.0.zip) or [Crosswalk 22 Android](https://download.01.org/crosswalk/releases/crosswalk/android/canary/22.52.557.0/crosswalk-22.52.557.0.zip) bundle or above.
```

Comment:
The app-tools can do this for sure, it's just my habit to download it manually :) I will change to use the app-tools instead.